apps/spruce/src/pages/task/taskTabs/TaskHistory/testData.ts

```diff
@@ -0,0 +1,170 @@
+import { TaskStatus } from "@evg-ui/lib/types/task";
+import { TaskHistoryTask } from "./types";
+
+export const tasks: TaskHistoryTask[] = [
+  {
+    id: "a",
+    activated: true,
+    displayStatus: TaskStatus.Succeeded,
+    execution: 0,
```

Comment:
We should consider showing execution on the details card

Reply:
I've asked Will if he we can include that on the designs!